FUSETOOLS-2454 - initial revisions for method selection dialog

bfitzpat opened the pull request:
Will update JIRA to include screen shots of design decisions for
discussion.

Signed-off-by: Brian Fitzpatrick <bfitzpat@redhat.com>

lhein commented on editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/globalconfiguration/beans/MethodSelectionDialog.java:
```diff
@@ -0,0 +1,236 @@
+	 *
+	 * @return object or null
+	 */
+	public Object getFirstResult() {
```
You could decide to use Selections.getFirstSelection(...) instead of rewriting that code here.

bfitzpat replied:
Didn't know about that! I will update. Thanks!

bfitzpat commented:
Fixed selection problem (it wasn't actually returning the selection because the control was disposed by the time it did), added double-click listener, and addressed @lhein feedback about re-using utility method.

bfitzpat commented:
testPR

bfitzpat commented:
argh. back to tests failing for no good reason

lhein commented:
testPR

bfitzpat commented:
testPR

bfitzpat commented:
PR build is failing because Eclipse on Jenkins says it can't find some schemas like http://camel.apache.org/schema/blueprint/camel-blueprint.xsd -- which is accessible so far as I can tell.
